packages/main/src/RPA/Images.py

```diff
@@ -243,7 +247,7 @@ def find_template_in_image(
         if not matches:
             raise ImageNotFoundError("No matches for given template")
 
-        # Convert region coördinates back to full-size coördinates
```

Comment:
Oh no, you removed my favourite stupid spelling: https://en.wiktionary.org/wiki/co%C3%B6rdinate#English

Reply:
Oh my :D I'll undo the change